modules/nw-create-load-balancer-service.adoc

```diff
@@ -54,8 +54,7 @@ spec:
 +
 [NOTE]
 ====
-To restrict traffic through the load balancer to specific IP addresses, it is recommended to use the `service.beta.kubernetes.io/load-balancer-source-ranges` annotation rather than setting the `loadBalancerSourceRanges` field.
-With the annotation, you can more easily migrate to the OpenShift API, which will be implemented in a future release.
+To restrict the traffic through the Load Balancer to specific IP addresses, it is recommended to use the `ingresscontroller` field `spec.endpointPublishingStrategy.loadBalancer.allowedSourceRanges`. Do not set the `loadBalancerSourceRanges` field.
```

Comment:
Recommend using "Ingress Controller" instead of `ingresscontroller`.

Reply:
Addressed